playbooks/common/openshift-cluster/upgrades/initialize_nodes_to_upgrade.yml

```
@@ -44,6 +44,6 @@
       ansible_become: "{{ g_sudo | default(omit) }}"
     when: item not in dedicated_etcds
     vars:
-      dedicated_etcds: "{{ groups['oo_etcd_to_config'] | difference(groups['oo_masters']) }}"
+      dedicated_etcds: "{{ groups['oo_etcd_to_config'] | difference( union(groups['oo_masters'],groups['oo_nodes_to_config']) ) }}"
```

```suggestion
      not_nodes: "{{ union(groups['oo_masters'] | list, groups['oo_nodes_to_config'] | list) }}"
      dedicated_etcds: "{{ groups['oo_etcd_to_config'] | difference(not_nodes) }}"
```
(didn't test this though)

---

`groups` returns lists.  Updated to correct jinja syntax.